ui/src/components/graph/graphNodeDetails.tsx

```diff
@@ -0,0 +1,87 @@
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const isFeature = (featureType:string) => {
+    return featureType === 'feathr_anchor_feature_v1' || featureType === 'feathr_derived_feature_v1'
+  }
+
+  useEffect(() => {
+    const fetchFeatureData = async () => {
+      setFeature(undefined);
+      if (nodeId && isFeature(featureType)) {
+        setLoading(true);
+        const data = await fetchFeature(project, nodeId);
+        setFeature(data);
+        setLoading(false);
+      }
+    };
+
+    fetchFeatureData();
+  }, [nodeId]);
+
+  return (
+    <>
+      {
+        loading
+          ? (<Spin indicator={ <LoadingOutlined style={ { fontSize: 24 } } spin /> } />)
+          : (<div style={ { margin: "2%" } }>
+            { feature?.attributes.transformation &&
+                <Card title="Transformation">
+                  { feature.attributes.transformation.transform_expr &&
+                      <p>transform_expr: { feature.attributes.transformation.transform_expr }</p> }
+                  { feature.attributes.transformation.filter &&
+                      <p>filter: { feature.attributes.transformation.filter }</p> }
+                  { feature.attributes.transformation.agg_func &&
+                      <p>agg_func: { feature.attributes.transformation.agg_func }</p> }
+                  { feature.attributes.transformation.limit &&
+                      <p>limit: { feature.attributes.transformation.limit }</p> }
+                  { feature.attributes.transformation.group_by &&
+                      <p>group_by: { feature.attributes.transformation.group_by }</p> }
+                  { feature.attributes.transformation.window &&
+                      <p>window: { feature.attributes.transformation.window }</p> }
+                  { feature.attributes.transformation.def_expr &&
+                      <p>def_expr: { feature.attributes.transformation.def_expr }</p> }
+                </Card>
+            }
+            { feature?.attributes.key && feature.attributes.key.length > 0 &&
+                <Card title="Key">
+                    <p>full_name: { feature.attributes.key[0].full_name }</p>
+                    <p>key_column: { feature.attributes.key[0].key_column }</p>
+                    <p>description: { feature.attributes.key[0].description }</p>
+                    <p>key_column_alias: { feature.attributes.key[0].key_column_alias }</p>
+                    <p>key_column_type: { feature.attributes.key[0].key_column_type }</p>
+                </Card>
+            }
+            { feature?.attributes.type &&
+                <Card title="Type">
+                    <p>dimension_type: { feature.attributes.type.dimension_type }</p>
+                    <p>tensor_category: { feature.attributes.type.tensor_category }</p>
+                    <p>type: { feature.attributes.type.type }</p>
+                    <p>val_type: { feature.attributes.type.val_type }</p>
```

Comment:
should we reuse this cards? this seems a duplication of the standalone detail page.

Reply:
The render of render details is somehow different in graph details page, the room in graph page is small so it's indented not to show all details information. For example, fields with N/A won't get displayed.

Current 3 cards design in feature details page is going to be revamped soon. So I am closing this as won't fix.